pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1911,6 +1911,32 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes[0].Field).To(Equal("fake.domain.devices.networkInterfaceMultiqueue"))
 		})
 
+		Context("VMI with network multiqueue", func() {
```

Comment:
I think this context fits a table, reducing layers and setup specs.
You have already done something similar in the e2e tests.

Reply:
done